content/sensu-go/5.16/reference/agent.md

```diff
@@ -972,6 +974,18 @@ deregistration-handler: "deregister"{{< /highlight >}}
 
 ### Keepalive configuration flags
 
+| keepalive-critical-timeout |      |
+--------------------|------
+description         | Number of seconds after a missing keepalive event until the agent is considered unresponsive by the Sensu backend to create a critical event. If the value is not `0`, it must be greater than or equal to `5` (in other words, the value cannot be `1`, `2`, `3`, or `4`).
```

Comment:
```suggestion
description         | Number of seconds after a missing keepalive event until the agent is considered unresponsive by the Sensu backend to create a critical event. Set to disabled (`0`) by default. If the value is not `0`, it must be greater than or equal to `5` (in other words, the value cannot be `1`, `2`, `3`, or `4`).
```

Reply:
> (in other words, the value cannot be `1`, `2`, `3`, or `4`)

This can probably be removed. The value cannot be negative either, so the preceding statement accounts for that.